bin/mergeq_ci.sh

```diff
@@ -77,6 +101,8 @@ target_branch=$2
 
 validate_parameters
 
+run_hook "before_ci_startup" # check_dependencies
```

Comment:
I'm not sure if this one is even necessary, the dependencies are needed for one of the other hooks, so they could happen then. I suppose it's good to have any errors ahead of time though.